src/test/java/org/testeditor/web/xtext/index/resources/IGlobalScopeResourceTest.xtend

```diff
@@ -19,12 +19,14 @@ import org.testeditor.web.xtext.index.serialization.EObjectDescriptionSerializer
 
 import static org.assertj.core.api.Assertions.assertThat
 import org.eclipse.xtext.naming.QualifiedName
+import org.junit.Ignore
 
 /**
  * Tests whether the REST interface can be successfully invoked via REST.
  * 
  * The test uses the Xtext language as illustrative example.
  */
+@Ignore
```

Comment:
OK, though I fixed this test somewhere "further up", see https://github.com/test-editor/xtext-index-backend/commit/490bb235f39333855b98fa225fd3c2ed163305bd

Reply:
I know, that's why I simply ignore it here (so the build does not break). It will be fixed in our integration branch